### DISCUSSION TITLE
[Projects Beta] Add field for issue number

### DISCUSSION THREAD

**Discussion by littlespex** [community/community#16014] [Projects and Issues] [answered]
When viewing a project table view, the title of the issue appears as a column, but there is no way to see the issue number. We often reference open issues by thier number. We also use the "Create a branch" feature in the Development sidebar, which creates branches prefixed with the issue number.

Would it be possible to add a new field for "Issue Number" to the standard list of available fields?

**Comment by bnsfa** [comment]
اعلانات

**Comment by littlespex** [comment, marked as answer]
Issue numbers now appear in the title field.
